resources/styleheets/main.css

```diff
@@ -0,0 +1,263 @@
+body {
+  padding: 0;
+  margin: 0;
+  height: 100%;
+  width: 100%;
+  background-color: #1B1717;
+  font-family: "Almarai", sans-serif;
+}
```

Comment:
you already have a style declared in _base.scss for body!!!
base is for declare the base attributes you want in the project and normally you point it as raw element:
```
body {
  // base style here
}

h1, h2, h3, h4, h5, h6 {
  // base style here
}

... and you continue
```
notice that i don't use a .class or a #id in the base, just raw elements